src/bot/bot.py

```diff
@@ -1,16 +1,18 @@
 from mmpy_bot import Bot, Settings
 from plugins import MyPlugin
 
+from depends import settings
+
 
 def init_bot():
```

Comment:
Пусть init_bot будет принимать settings. 
```suggestion
def init_bot(settings: Settings = Provide[Container.settings]):
```